### ISSUE TITLE
Fix a11y issue: Create instance modal and aria-hidden not being removed

### ISSUE THREAD

**Issue by christiemolloy** [bf2fc6cc711aee1a0c2a/kas-ui#102]
Regarding the issue with aria-hidden not being removed, I noticed that when I click the Generate Credential button in the mock UI, that this attribute is removed when the modal closes, as expected. The way the Generate Credential modal is implemented matches my expectation for how the modal should be implemented, with one file that includes both the toggle that displays the modal, and the modal itself. I'm not sure if this is related to why the aria-hidden attribute works as expected or not. 

I don't think this is a PF issue, but rather an issue with how we're handling the modal. I'd suggest handling this with a separate issue/PR.

_Originally posted by @jgiardino in https://github.com/bf2fc6cc711aee1a0c2a/mk-ui-frontend/issues/92#issuecomment-742828629_
